cmake/CMakeLists.txt

```diff
@@ -2157,65 +1365,109 @@ if (onnxruntime_USE_WINML)
         FATAL_ERROR
         "Option onnxruntime_USE_WINML can only be used when onnxruntime_BUILD_SHARED_LIB is also enabled")
   endif()
-  include(winml.cmake)
+  list(APPEND ONNXRUNTIME_TARGETS winml)
```

Comment:
nit: I don't think there is a target named "winml" in winml.cmake. the variable "ONNXRUNTIME_TARGETS" sounds like it should contain target names, but it looks like it is actually cmake file basenames. so maybe rename the variable or make winml.cmake define a consistently named target?

Reply:
How about ONNXRUNTIME_CMAKE_FILES?

Reply:
sure that's good